spec/ics-3-connection-semantics/README.md

```diff
@@ -0,0 +1,145 @@
+---
+ics: 3
+title: Connection Semantics
+stage: draft
+category: ibc-core
+requires: 2, 6, 10, 23
+required-by: 4
+author: Christopher Goes <cwgoes@tendermint.com>, Juwoon Yun <joon@tendermint.com>
+created: 2019-03-07
+modified: 2019-03-29
+---
+
+## Synopsis
+
+This standards document describes the abstraction of an IBC *connection*: two stateful objects on two separate chains, each tracking the consensus state of the other chain, facilitating cross-chain substate verification. Protocols for safely establishing a connection between two chains, verifying relayed updates (headers) to the consensus state tracked by a connection, cleanly closing a connection, and closing a connection due to detected equivocation are described.
+
+## Specification
+
+### Motivation
+
+The core IBC protocol provides *authorization* and *ordering* semantics for packets: guarantees, respectively, that packets have been committed on the sending blockchain (and according state transitions executed, such as escrowing tokens), and that they have been committed exactly once in a particular order and can be delivered exactly once in that same order. The *connection* abstraction, specified in this standard, defines the *authorization* semantics of IBC (ordering semantics are left to [ICS 4: Channel Semantics](../spec/ics-4-channel-sematics)).
+
+### Desired Properties
+
+- Implementing blockchains should be able to safely allow untrusted users to open and update connections.
```

Comment:
Generally, it's referring to a user who is paying for computation / storage (via gas or something) but is otherwise untrusted. Example users:
- End-users (humans) with keys
- Smart contracts
- Other modules (e.g. in the SDK)

Thinking about where best to explain this...

Reply:
I've added essentially the above under "Definitions". That seems OK for now, but maybe this should transition to a separate "common definitions" location in the future.